docs/index.md

```diff
@@ -1179,6 +1179,47 @@ app/components
 </div>
 ```
 
+##### Translations
```

Comment:
Can we mark this as experimental similar to how we've marked V2 as experimental?

The goal is to include this for V3 but don't want to commit to a specific API/behavior until then.